CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# CHANGELOG
+Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
+## [ UNRELEASED ]
+### Added
+- Github workflow for changelog verification ([#4085](https://github.com/opensearch-project/OpenSearch/pull/4085))
+- Your change here ([#PR_NUMBER](https://github.com/opensearch-project/OpenSearch/pull/PR_NUMBER))
```

Comment:
this shouldn't be here as otherwise this will surely end up in the final changelog as nobody will bother to remove it. you already have your first entry as an example, that should be enough